testing/rclone/APKBUILD

```diff
@@ -0,0 +1,98 @@
+# Contributor: tcely <rclone+aports@tcely.33mail.com>
+# Maintainer: tcely <rclone+aports@tcely.33mail.com>
+_pkgname="rclone"
+pkgname="${_pkgname}"
+pkgver="1.48.0"
+pkgrel=0
+pkgdesc="rsync for cloud storage"
+url="https://rclone.org/"
+arch="all"
+license="MIT"
+options=""
+depends=""
+makedepends="bash git go perl"
```

Comment:
They are used in the Makefile

Reply:
I realize that.
Why are you using the Makefile?
That was another part of my review that was um... marked resolved without any interaction.
The Makefile is extraordinarily strange (read it!) and is wholly unnecessary.
I even gave specific actionable push-single-button-to-apply replacement snippets.
If you had any specific reason to use the Makefile over standard golang build procedures, I'd imagine you would have responded to that part of the review with it...

As-is, you're introducing a potential failure condition and requesting additional makedepends for no noticeable reason.

Reply:
I'm preferring to use the upstream build process over replacing it.

Reply:
The upstream build process is documented [here](https://rclone.org/install/).
Quotes for your convenience:

> Make sure you have at least Go 1.7 installed. Download go if necessary. The latest release is recommended. Then
> ```
> git clone https://github.com/ncw/rclone.git
> cd rclone
> go build
> ./rclone version
> ```
> You can also build and install rclone in the GOPATH (which defaults to \~/go) with:
> `go get -u -v github.com/ncw/rclone`
> and this will build the binary in $GOPATH/bin (\~/go/bin/rclone by default) after downloading the source to $GOPATH/src/github.com/ncw/rclone (~/go/src/github.com/ncw/rclone by default).

Note that the `Makefile` is never mentioned.

Further, do consider the standard build process for a given language.
What would you do if an upstream had a standard CMake setup, but also a `./build` script that pulled in python, ruby, perl, go and rust, with no additional functionality?

Reply:
If upstream provided documentation that did not mention `build.sh` and listed manual steps to take, I would also choose `build.sh` in that case as it provides the opportunity for upstream to improve their build process without interaction from downstream.